scripts/deployment/deploy_18_change_managers.js

```diff
@@ -32,7 +32,7 @@ async function main() {
 
     // Transaction signing and execution
     // 18. EOA to change the manager of ServiceRegistryTokenUtility to ServiceManagerToken calling `changeManager(ServiceManagerToken)`;
-    console.log("You are signing the following transaction: componentRegistry.connect(EOA).changeManager()");
+    console.log("You are signing the following transaction: serviceRegistryTokenUtility.connect(EOA).changeManager()");
```

Comment:
```suggestion
    console.log("You are signing the following transaction: serviceManagerTokenAddress.connect(EOA).changeManager()");
```

Reply:
same below please

Reply:
that's not really the case, but I understood what you meant

Reply:
yes, sorry. You are right! Lgt